hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.test.tsx

```diff
@@ -151,6 +155,46 @@ describe("Dashboard Page", () => {
             expect(mockedGet).toHaveBeenCalledWith("/api/event/teams/team/");
         });
     });
+
+    it("Removes orders when cancel order button is clicked", async () => {
+        const pendingOrderDetailUri = "/api/hardware/orders/4";
+        const pendingOrderResponse = makeMockApiResponse(mockPendingOrdersInTable[1]);
+        when(mockedPatch)
+            .calledWith(pendingOrderDetailUri, { status: "Cancelled" })
+            .mockResolvedValueOnce(pendingOrderResponse);
+
+        const store = makeStoreWithEntities({
+            pendingOrders: mockPendingOrdersInTable,
+        });
+
+        const { getByTestId, queryByText, getByText } = render(<Dashboard />, {
+            store,
+        });
+
+        await waitFor(() => {
+            mockPendingOrdersInTable.forEach((order) => {
+                expect(getByText(`Order #${order.id}`)).toBeInTheDocument();
+            });
+        });
+
+        const orderItem = getByTestId(
```

Comment:
just fyi, this data test id is on the order table, not an order item (an order item is a hardware item you put in the order). 
Naming this table "pending-order-item" may cause confusion because in the backend we have an OrderItem table that keeps track of order items which are different from orders.

Reply:
To avoid this confusion we may want to rename `<PendingTable/>` to `<PendingTables/>` since all the pending orders are in that component. Same with CheckedOutTable, but that can be a separate ticket.